exporter/datadogexporter/traces_exporter.go

```diff
@@ -213,7 +213,7 @@ func newTraceAgentConfig(ctx context.Context, params exporter.Settings, cfg *Con
 	acfg.OTLPReceiver.SpanNameAsResourceName = cfg.Traces.SpanNameAsResourceName
 	acfg.Endpoints[0].APIKey = string(cfg.API.Key)
 	acfg.Ignore["resource"] = cfg.Traces.IgnoreResources
-	acfg.ReceiverPort = 0 // disable HTTP receiver
```

Comment:
Do we need to set both? Or `ReceiverEnabled` is enough

Reply:
`ReceiverEnabled:false` is enough, and handles both TPC and UDS listeners. `ReceiverPort:0` only disabled TCP listener.